server/pa.js

```diff
@@ -2,12 +2,43 @@
 const Validator = require('express-validator')
 
 // In-house dependencies
-const { helpers, twilioHelpers } = require('brave-alert-lib')
+const { helpers, googleHelpers, twilioHelpers } = require('brave-alert-lib')
 const db = require('./db/db')
 
 const paApiKeys = [helpers.getEnvVar('PA_API_KEY_PRIMARY'), helpers.getEnvVar('PA_API_KEY_SECONDARY')]
 const paPasswords = [helpers.getEnvVar('PA_PASSWORD_PRIMARY'), helpers.getEnvVar('PA_PASSWORD_SECONDARY')]
 
+const validateGetGoogleTokens = Validator.body(['authCode']).trim().notEmpty()
```

Comment:
Could you please also add some checks for these validate functions? You can probably see how this has been done elsewhere in the integration tests folder.

Reply:
I don't actually see any examples of testing for these functions being done in BraveSensor, and am not sure how to go about it. I'll leave this out unless someone can give me more direction on how to do this... But I have trust that the Validator object is thoroughly tested where it is defined.

Reply:
I agree that it isn't obvious. For future reference, the validator tests can be done in the integration tests. For example, https://github.com/bravetechnologycoop/BraveSensor/blob/main/server/test/integration/dashboardTest/submitEditClientTest.js#L821-L825